specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/examples/ManagedHsm_CreateOrUpdate.json

```diff
@@ -81,6 +81,15 @@
           "Environment": "dogfood"
         }
       }
+    },
+    "default": {
+      "headers": {},
+      "body": {
```

Comment:
suggest to use reference `common-types/resource-management/v2/types.json#/definitions/ErrorResponse`. Sample: https://github.com/Azure/azure-rest-api-specs/blob/5bb1209e89f8eaf6cce54b80f0df0c1e1f39759a/specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-03-01-preview/authorization-RoleAssignmentsCalls.json#L65

Reply:
The sample is the swagger specification file and we use the reference in our files as well:
> https://github.com/Azure/azure-rest-api-specs/blob/master/specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/managedHsm.json#L148

For the sample repos, they use the same example schema:
> https://github.com/Azure/azure-rest-api-specs/blob/5bb1209e89f8eaf6cce54b80f0df0c1e1f39759a/specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-03-01-preview/examples/RoleAssignments_DeleteById.json#L21

Reply:
oh I see. These are all example files. Fixing examples are not required by S360.